public/app/features/users/TokenRevokedModal.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Button, Modal } from '@grafana/ui';
+import config from 'app/core/config';
+
+interface Props {}
+
+interface State {}
+
+export class TokenRevokedModal extends React.Component<Props, State> {
+  redirectToLogin() {
+    window.location.href = config.appSubUrl + '/';
+  }
+
+  render() {
+    return (
+      <Modal title="Token revoked" isOpen={true} onDismiss={this.redirectToLogin} onClickBackdrop={() => {}}>
```

Comment:
I intentionally added the `() => {}` to avoid the user clicking outside the modal.

Is that okay in terms of UX? @lukasztyrala 
Should I define a method like `doNothing` and refer it to make it more explicit? @Clarity-89 

Thanks!!